lib/resty/rediscluster.lua

```diff
@@ -30,14 +30,17 @@ local DEFAULT_SEND_TIMEOUT = 1000
 local DEFAULT_READ_TIMEOUT = 1000
 
 local function parse_key(key_str)
-    local left_tag_single_index = string_find(key_str, "{", 0)
-    local right_tag_single_index = string_find(key_str, "}", 0)
-    if left_tag_single_index and right_tag_single_index then
+    local left_tag_single_index = string_find(key_str, "{", 1)
```

Comment:
it's better to add comments to explain simply and link [Redis HashSlot](https://github.com/redis/redis/blob/8a05f0092b0e291498b8fdb8dd93355467ceab25/src/cluster.c#L30-L49) implementation and [redis.io/docs/latest/operate/oss_and_stack/reference/cluster-spec/#hash-tags](https://redis.io/docs/latest/operate/oss_and_stack/reference/cluster-spec/#hash-tags)
before the function `local function parse_key(key_str)`

Reply:
done